tests/expense.test.js

```diff
@@ -188,6 +188,7 @@ describe('Expense', () => {
         });
 
       const response = await api
+      // eslint-disable-next-line max-len
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find the way to solve it without `eslint-disable`.

Reply:
And you missed this comment ☝️  It is still actual.